readthedocs/api/v2/views/model_views.py

```diff
@@ -272,6 +272,25 @@ def retrieve(self, *args, **kwargs):
                     )
         return Response(data)
 
+    @decorators.action(
+        detail=True,
+        permission_classes=[permissions.IsAdminUser],
+        methods=['post'],
+    )
+    def restart(self, request, **kwargs):
```

Comment:
I'd call this something like "clean_commands", just "clean", "reset" or similar since "restart" suggests that the build will be re-triggered.

Reply:
I like reset